science/providers/python_build_standalone.py

```diff
@@ -49,18 +52,61 @@ def with_fingerprint(self, fingerprint: Fingerprint) -> FingerprintedAsset:
 
 
 @dataclass(frozen=True)
-class PythonBuildStandalone(Provider):
-    """Provides *pre-built* CPython distributions from the Python Standalone Builds project.
+class Config:
+    version: str = dataclasses.field(
+        metadata=help_doc(
+            """The CPython version to select.
 
-    All science platforms are supported for Python 3 minor versions >= 8. Python Standalone Builds
-    does not provide all patch versions; so you should check their releases if you wish to pin down
-    to the patch level. See here: https://github.com/indygreg/python-build-standalone/releases
+            Can be either in `<major>.<minor>` form or else fully specified as
+            `<major>.<minor>.<patch>`.
+
+            ```{caution}
+            Python Standalone Builds does not provide all patch versions; so you should check
+            their releases if you wish to pin down to the patch level.
+            See here: https://github.com/indygreg/python-build-standalone/releases
+            ```
+            """
+        )
+    )
+    flavor: str = dataclasses.field(
+        default="install_only",
+        metadata=help_doc(
+            """The flavor of the Python Standalone Builds release to use.
+
+            Currently only accepts 'install_only' which is the default.
+            """
+        ),
+    )
+    release: str | None = dataclasses.field(
+        default=None,
+        metadata=help_doc(
```

Comment:
Think it's worth also mentioning an example release and citing the same URL as above?
https://github.com/indygreg/python-build-standalone/releases

e.g `20230507`

I got burned by this my first time around.

Reply:
Definitely. Done.

I re-worked the field sections so that they are linkable. The screen shot is updated - note the link tree in the contents siderbar summarizing this new heading structure.